### DISCUSSION TITLE
diagonal PASEF (midia, synchro, slice)

### DISCUSSION THREAD

**Discussion by ghost** [vdemichev/DiaNN#1030] [General]
Hello DIA-NN team, 

As usual thanks for your work and time. Straight to the point: There has been a lot of buzz about the so called diagonal PASEF methods recently. Can DIA-NN (recent beta versions) handle these types of data?

The --tims -scan command enables processing of slice PASEF data in the beta version I currently use (1.8.2 beta 27), can the same command be used for data acquired in midia or synchro PASEF modes? Or is something else currently being implemented? I am really keen to try some diagonal PASEF (synchro) on our timsToF but I am not sure if I can process the data with software I have available!

Cheers,

Andrew Frey

**Comment by vdemichev** [comment]
Hi Andrew,

Yes, --tims-scan works for the scanning modes, although currently DIA-NN does not really deconvolute the Q1 dimension, i.e. the performance will eventually become better.

Best,
Vadim
